packet-ethereum-disc.c

```diff
@@ -426,43 +466,30 @@ static int process_findnode_msg(tvbuff_t *packet_tvb,
   return TRUE;
 }
 
-/**
- * Processes a NODES packet.
- *
- * @param packet_tvb The buffer representing only the packet payload (excluding the message wrapper).
- * @param packet_tree The protocol tree representing the packet.
- * @param pinfo Packet
- * @param rlp The RLP element pointing to the list representing the packet.
- * @param st A ready-to-use statistics struct to populate.
- * @param conv A ready-to-use conversation struct (retrieved or initialized).
- * @param efdata A ready-to-use enhanced frame data struct.
- * @return TRUE if processing was successful; FALSE otherwise.
- */
-static int process_nodes_msg(tvbuff_t *packet_tvb,
-                             proto_tree *packet_tree,
-                             packet_info *pinfo,
-                             rlp_element_t *rlp,
-                             ethereum_disc_stat_t *st,
-                             ethereum_disc_conv_t *conv,
-                             ethereum_disc_enhanced_data_t *efdata _U_) {
-  proto_tree *parent;
+static void process_nodes_list(tvbuff_t *packet_tvb,
```

Comment:
Could we rename this function to something like `decode_nodes_list`? It's not explicitly stated, but the convention I've used is to prefix functions that a wire message/packet with `process_`. AFAIU, this is more of a helper function.

Reply:
Makes sense